AutomatedTesting/Gem/PythonTests/atom_renderer/atom_utils/benchmark_utils.py

```diff
@@ -61,6 +61,25 @@ def capture_pass_timestamp(self, frame_number):
             general.log('Failed to capture pass timestamps.')
         return self.capturedData
 
+    def capture_cpu_frame_time(self, frame_number):
+        """
+        Capture CPU frame times and block further execution until it has been written to the disk.
+        """
+        self.handler = azlmbr.atom.ProfilingCaptureNotificationBusHandler()
+        self.handler.connect()
+        self.handler.add_callback('OnCaptureCpuFrameTimeFinished', self.on_data_captured)
+
+        self.done = False
+        self.capturedData = False
+        success = azlmbr.atom.ProfilingCaptureRequestBus(
+            azlmbr.bus.Broadcast, "CaptureCpuFrameTime", f'{self.output_path}/cpu_frame{frame_number}_time.json')
+        if success:
+            self.wait_until_data()
+            general.log('CPU frame time captured.')
+        else:
+            general.log('Failed to capture CPU frame time.')
+        return self.capturedData
```

Comment:
why do we always return `False` for this function? was trying to figure that part out because `self.capturedData` gets explicitly set to `False` on line 73 then we return it - plus it's already an attribute of the `BenchmarkHelper` object so no need to return it if we're setting it on the object itself already `self.capturedData` 
Functions return `None` by default if no return value is set, which would be functionally equivalent in most use cases as returning `False` also

Reply:
`self.capturedData` gets set to True when `self.on_data_captured` gets called as a callback to the 'OnCaptureCpuFrameTimeFinished' event, and `self.wait_until_data()` should block the return of `self.capturedData` until `self.done` is set to True in `self.on_data_captured()`.